pycodestyle.py

```diff
@@ -822,7 +822,7 @@ def whitespace_before_parameters(logical_line, tokens):
             not keyword.iskeyword(prev_text) and
             # 'match' and 'case' are only soft keywords
             (
-                sys.version_info <= (3, 10) or
+                sys.version_info < (3, 10) or
```

Comment:
it should really be 3, 9 -- where the function was introduced (this will make cleanup maintenance earlier and easier in the future as well)

Reply:
Didn't know that. Updated the PR